pkg/inference/scheduling/scheduler.go

```diff
@@ -194,14 +198,17 @@ func (s *Scheduler) handleOpenAIInference(w http.ResponseWriter, r *http.Request
 
 	// Check if the shared model manager has the requested model available.
 	if !backend.UsesExternalModelManagement() {
-		if _, err := s.modelManager.GetModel(request.Model); err != nil {
+		model, err := s.modelManager.GetModel(request.Model)
+		if err != nil {
 			if errors.Is(err, distribution.ErrModelNotFound) {
 				http.Error(w, err.Error(), http.StatusNotFound)
 			} else {
 				http.Error(w, "model unavailable", http.StatusInternalServerError)
 			}
 			return
 		}
+		// Non-blocking call to track the model usage.
+		s.tracker.TrackModel(model)
```

Comment:
Maybe a stupid question, but should we disable this in Docker Desktop?  Otherwise I'm not sure how we would delineate Docker CE statistics.

Reply:
Let's set another User-Agent for Docker Desktop. 😁 
Done in https://github.com/docker/model-runner/pull/58/commits/d24babea539d23e9cc2f093f05292bbe0d55ef9a, as well as a way to disable tracking programmatically.